src/com/Jacksonnn/DCCore/StaffUtils/StaffChannels/onChatEvent.java

```diff
@@ -30,7 +30,7 @@ public void onPlayerChat(AsyncPlayerChatEvent e) {
             Collection<? extends Player> onlinePlayers = Bukkit.getServer().getOnlinePlayers();
 
             for (Player onlinePlayer : onlinePlayers) {
-                if (onlinePlayer.hasPermission("DCCore.staffchats.HoS")) {
+                if (onlinePlayer.hasPermission("DCCore.staffchats.HOS")) {
```

Comment:
It would probably be best to keep it as HoS for the sake of reducing the labor of updating

Reply:
I mean the only thing needing updating is the permission that 4 people have, doesn’t seem too much of a hassle to me but